cypress/integration/publication_description.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Publication: Description', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.server();
+  });
+
+  it('The user should be able to fill out the form for book type', () => {
+    cy.mocklogin();
+
+    cy.get('[data-testid=new-publication-button]').click({ force: true });
+    cy.get('[data-testid=new-schema-button]').click({ force: true });
```

Comment:
kanskje legge inn en TODO om å endre denne testen når vi har fått DOI-link på plass?

Reply:
Trenger vi det, om vi er flinke til å følge opp testene for koden vi rører? #optimist